ui/main/src/app/components/navbar/navbar.component.spec.ts

```diff
@@ -90,18 +94,48 @@ describe('NavbarComponent', () => {
         expect(component.limitSize).toBe(undefined);
     });
 
-    it('should create with a configuration setted', () => {
-        defineFakeState(MODE.HAS_CONFIG);   
+    it('should create with the custom logo configuration setted to true', () => {
```

Comment:
`<grammar rigorist>` _set_ may be used instead of _setted_`</gramma rigorist>`

Reply:
fixed

Reply:
fixed